Library/Homebrew/cleanup.rb

```diff
@@ -23,10 +24,27 @@ def self.disk_cleanup_size
       @disk_cleanup_size
     end
 
+    def self.add_unremovable_keg(keg)
+      @unremovable_kegs << keg
+    end
+
+    def self.unremovable_kegs
+      @unremovable_kegs
+    end
```

Comment:
I don't think these helper methods are needed; just write directly to the instance variable.